README.md

```diff
@@ -1,7 +1,7 @@
 1. Replace `<your_account>` with your Github username in the link
-    - [DEMO LINK](https://<your_account>.github.io/js_get_data_DOM/)
+    - [DEMO LINK](https://tarasavage.github.io/js_get_data_DOM/)
```

Comment:
your link to demo doesn't work, did you do deploy?

Reply:
yes, I don't know why it's not working

Reply:
I think you forgot to deploy your code. Please, deploy it and add the link to the description
<img width="1680" alt="image" src="https://user-images.githubusercontent.com/99403198/211155157-56069202-c7d5-439f-b8f2-27fdb4175d5c.png">